Sources/Networking.swift

```diff
@@ -22,6 +22,9 @@ public protocol URLSessionTaskNDT7 {
 }
 
 extension URLSession: URLSessionNDT7 {
+    public func ndt7DataTask(with request: URLRequest, completionHandler: @escaping @Sendable  (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask {
```

Comment:
```suggestion
  public func ndt7DataTask(with request: URLRequest, completionHandler: @escaping @Sendable  (Data?, URLResponse?, Error?) -> Void) -> URLSessionDataTask {
```

Reply:
I think this repo uses 4 space tabs 🤷

Reply:
Ah that works. The next two lines are only indented with two space tabs though.